app/gui2/src/util/array.ts

```diff
@@ -36,6 +36,9 @@ export function partitionPoint<T>(
 }
 
 /** Index into an array using specified index. When the index is nullable, returns undefined. */
-export function tryGetIndex<T>(arr: T[], index: number | undefined | null): T | undefined {
-  return index == null ? undefined : arr[index]
+export function tryGetIndex<T>(
+  arr: T[] | undefined | null,
```

Comment:
are we no longer using `Opt<>`?

Reply:
Yeah, we should, at least for the arguments. It has its problems on return value, but here it is ok.